Random 500 Errors in circle CI tests

We seem to intermittently see errors as follows in the circle CI tests.

~~Could this be that the service isn't ready prior to running the test?~~

```
_________ ERROR at setup of test_there_are_ten_most_popular_extensions _________

request = <SubRequest '_verify_url' for <Function 'test_there_are_ten_most_popular_extensions'>>
base_url = 'http://127.0.0.1'

    @pytest.fixture(scope='session', autouse=True)
    def _verify_url(request, base_url):
        """Verifies the base URL"""
        verify = request.config.option.verify_base_url
        if base_url and verify:
            response = requests.get(base_url, timeout=10)
            if not response.status_code == requests.codes.ok:
                raise pytest.UsageError(
                    'Base URL failed verification!'
                    '\nURL: {0}, Response status code: {1.status_code}'
>                   '\nResponse headers: {1.headers}'.format(base_url, response))
E               UsageError: Base URL failed verification!
E               URL: http://127.0.0.1, Response status code: 500
E               Response headers: {'ETag': '"6e6be9f80c675e14d4f487c43dc5dd49"', 'Date': 'Mon, 05 Sep 2016 17:20:36 GMT', 'Vary': 'X-Mobile, User-Agent', 'x-content-type-options': 'nosniff', 'Content-Type': 'text/html', 'Server': 'nginx/1.9.9', 'X-Frame-Options': 'DENY', 'Transfer-Encoding': 'chunked', 'x-xss-protection': '1; mode=block', 'Connection': 'keep-alive'}
```

--- a/scripts/setup-docker.sh
+++ b/scripts/setup-docker.sh
@@ -1,5 +1,10 @@
 #!/bin/bash -x
 
+# Update deps to avoid missing dependencies
+# from the UI tests when a PR adds new dependency
+# that isn't yet built into the container.
+make update_deps
+
 # initialize_db:
 python manage.py reset_db --noinput
 python manage.py syncdb --noinput

